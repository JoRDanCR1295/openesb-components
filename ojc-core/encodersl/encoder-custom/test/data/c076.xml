<?xml version="1.0" encoding="UTF-8"?><root xmlns="http://xml.netbeans.org/schema/test" xsi:schemaLocation="${SCHEMA_LOCATION}" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><Ident>PriceReport001</Ident><DateCre>2008-08-08T09:30:47</DateCre><NavDate>2008-08-08</NavDate><Isin>'LU1234567890'</Isin><TP>NAVL</TP><Mnt>1100</Mnt><Ccy>EUR</Ccy></root>