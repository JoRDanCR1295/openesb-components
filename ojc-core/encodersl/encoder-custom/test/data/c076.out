PriceReport001,2008-08-08T09:30:47,2008-08-08,'LU1234567890',NAVL,1100,EUR